support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -378,7 +383,8 @@ ControlSystems:
       AverageTimescaleFraction: 0.25
       Average0thDeriv: false
     Controller:
-      UpdateFraction: 0.03
+    #Changed UpdateFraction from 0.03 to 0.3 to increase run speed
```

Comment:
Maybe @knelli2 can suggest a good wording for this comment. Either way, align the comment with the line below and add a space after `#`.